src/lab14.py

```diff
@@ -36,6 +36,7 @@
 from lab10 import COOKIE_JAR, URL
 from lab11 import FONTS, NAMED_COLORS, get_font, linespace, paint_tree
 from lab11 import parse_color, parse_blend_mode
+from lab11 import BlockLayout, DocumentLayout, TextLayout, LineLayout, InputLayout
```

Comment:
lab13 patches BlockLayout; is this correct?

Reply:
Good catch